### PR TITLE
Helmfile deploying on dev - notify-document-download

### PR DESCRIPTION
**Pull request by P0NDER0SA**
## What are you changing?

Updating the helmfile code to deploy the document-download-api portion.  Secrets are being pulled from AWS Secrets Manager and the deployment is working on DEV.  Needs more investigation into whether the components are working as they should!

## Provide some background on the changes

https://app.zenhub.com/workspaces/notify-planning-core-6411dfb7c95fb80014e0cab0/issues/gh/cds-snc/notification-planning-core/450

## After merging this PR

- [ ] I have verified that the tests / deployment actions succeeded
- [ ] I have verified that any affected pods were restarted successfully
- [ ] I have verified that I can still log into [Notify production](https://notification.canada.ca)
- [ ] I have verified that the smoke tests still pass on production
- [ ] I have communicated the release in the #notify Slack channel.

### CODE REVIEW

File: helmfile/charts/notify-document-download/templates/NOTES.txt [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  {{- end }}
+{{- end }}
+{{- else if contains "NodePort" .Values.service.type }}
+  export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ include "notify-document-download.fullname" . }})
+  export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
+  echo http://$NODE_IP:$NODE_PORT
+{{- else if contains "LoadBalancer" .Values.service.type }}
+     NOTE: It may take a few minutes for the LoadBalancer IP to be available.
+           You can watch the status of by running 'kubectl get --namespace {{ .Release.Namespace }} svc -w {{ include "notify-document-download.fullname" . }}'
+  export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ include "notify-document-download.fullname" . }} --template "{{"{{ range (index .status.loadBalancer.ingress 0) }}{{.}}{{ end }}"}}")
```
[the remark this answers is not in the document]
Comment (P0NDER0SA) [reply helmfile/charts/notify-document-download/templates/NOTES.txt:15]: it's just notes!

File: helmfile/charts/notify-document-download/templates/NOTES.txt [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  echo http://$SERVICE_IP:{{ .Values.service.port }}
+{{- else if contains "ClusterIP" .Values.service.type }}
+  export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app.kubernetes.io/name={{ include "notify-document-download.name" . }},app.kubernetes.io/instance={{ .Release.Name }}" -o jsonpath="{.items[0].metadata.name}")
+  export CONTAINER_PORT=$(kubectl get pod --namespace {{ .Release.Namespace }} $POD_NAME -o jsonpath="{.spec.containers[0].ports[0].containerPort}")
+  echo "Visit http://127.0.0.1:8080 to use your application"
```
[the remark this answers is not in the document]
Comment (P0NDER0SA) [reply helmfile/charts/notify-document-download/templates/NOTES.txt:20]: notes

File: helmfile/charts/notify-document-download/templates/NOTES.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (P0NDER0SA) [reply helmfile/charts/notify-document-download/templates/NOTES.txt:15]: again, notes

File: helmfile/charts/notify-documentation/templates/tests/test-connection.yaml
Comment (ben851) [inline helmfile/charts/notify-documentation/templates/tests/test-connection.yaml:1]: Same Q

Comment (P0NDER0SA) [reply helmfile/charts/notify-documentation/templates/tests/test-connection.yaml:1]: removed them both. they are residual from our helmsessions.  probably included during a bootstrap

**Review by ben851 — APPROVED** [review]
LGTM

File: helmfile/charts/notify-admin/templates/NOTES.txt [hunk 3]
```diff
@@ -6,16 +6,16 @@
...
+  export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ include "admin.fullname" . }})
   export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
   echo http://$NODE_IP:$NODE_PORT
 {{- else if contains "LoadBalancer" .Values.service.type }}
      NOTE: It may take a few minutes for the LoadBalancer IP to be available.
-           You can watch the status of by running 'kubectl get --namespace {{ .Release.Namespace }} svc -w {{ include "notify.fullname" . }}'
-  export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ include "notify.fullname" . }} --template "{{"{{ range (index .status.loadBalancer.ingress 0) }}{{.}}{{ end }}"}}")
+           You can watch the status of by running 'kubectl get --namespace {{ .Release.Namespace }} svc -w {{ include "admin.fullname" . }}'
+  export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ include "admin.fullname" . }} --template "{{"{{ range (index .status.loadBalancer.ingress 0) }}{{.}}{{ end }}"}}")
```
[the remark this answers is not in the document]
Comment (ben851) [reply helmfile/charts/notify-admin/templates/NOTES.txt:15]: Notes only

File: helmfile/charts/notify-document-download/templates/NOTES.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (ben851) [reply helmfile/charts/notify-document-download/templates/NOTES.txt:20]: Notes only

**Pull request merged**